talk/owt/sdk/p2p/p2ppeerconnectionchannel.cc

```diff
@@ -1091,7 +1126,6 @@ void P2PPeerConnectionChannel::DrainPendingStreams() {
       if (stream->Source().video == owt::base::VideoSourceInfo::kScreenCast) {
         video_track_source = "screen-cast";
       }
-      RTC_CHECK(temp_pc_);
```

Comment:
@davidtag I removed this because it's unnecessary. We're already inside the temp_pc_ case which means it's alive and it's being kept alive by our ref.